src/lib.rs

```diff
@@ -1,10 +1,93 @@
+//! `logd-logger`
```

Comment:
Theres a crate that generates the Readme out of `lib.rs`. Can we use that in order to not maintain the same doc twice?

Reply:
There is a cargo plugin that does the trick. Should we use that? [cargo-sync-readme](https://crates.io/crates/cargo-sync-readme)

Reply:
Sounds like a plan. Go for it!